.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,14 @@
+name: pre-commit
+
+on:
+  push:
+
+jobs:
```

Comment:
https://github.com/s0undt3ch/pytest-salt-factories/blob/master/.github/workflows/testing.yml#L16

Reply:
Is this better?

Reply:
Think so